src/workflows/airtable/activities.ts

```diff
@@ -61,25 +68,27 @@ export async function findRecordWithEmail(
 }
 
 export async function getCreativeWallet(recId: string) {
-  const base = (await getConnectionAirTable()).base("appBwrlSCBQDC9UCV");
+  //const base = (await getConnectionAirTable()).base("appBwrlSCBQDC9UCV");
+  const base = (await getConnectionAirTable()).base(process.env.SMART_CONTRACT_AIRTABLE_BASE!);
   /*const base = (await getConnectionAirTable()).base(
     `${process.env.GROWTH_MASTER_AIRTABLE_BASE}`
   );*/
 
-  const result = await base<ICreativesMaster>("Creatives").find(recId);
+  const result = await base<ICreativesMaster>("Creatives Growth Master").find(recId);
```

Comment:
let's have ENVs for these and use their IDs rather their names

Reply:
@wasif1024 are we going to change these to ENVs or IDs so the name change doesn't affect them?